File: app/src/main/java/com/example/driverjaben/entrypoint/SplashActivity.java
package com.example.driverjaben.entrypoint;

import android.content.Intent;
import android.os.Handler;
import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.example.driverjaben.R;
import com.example.driverjaben.driver.loginandsignup.EnterPhone;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        ActionBar actionBar = getSupportActionBar();
        actionBar.hide();

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent intent = new Intent(SplashActivity.this, EnterPhone.class);
                startActivity(intent);
                finish();
            }
        }, 1000);
    }
}

File: app/src/main/java/com/example/driverjaben/entrypoint/SplashActivity.kt
package com.example.driverjaben.entrypoint

import android.content.Intent
import android.os.Handler
import android.support.v7.app.ActionBar
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

import com.example.driverjaben.R

class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        supportActionBar!!.hide()

        Handler().postDelayed({
            val intent = Intent(this@SplashActivity, IsLoggedIn::class.java)
            startActivity(intent)
            finish()
        }, 1000)
    }
}
